try
	a=1
	throw "foo"
	a=2
catch e
	print e
end
assert a==1
